.github/workflows/auto-label.yml

```
@@ -0,0 +1,23 @@
+name: Auto Label
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)